ArcGISRuntimeSDKQt_QMLSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,374 @@
+                        bufferFeatures();
+                    }
+                }
+            }
+        }
+
+        GraphicsOverlay {
+            id: bufferGraphicsOverlay
+            renderer: SimpleRenderer {
+                SimpleFillSymbol {
+                    style: Enums.SimpleFillSymbolStyleForwardDiagonal
+                    color: "red"
+                    outline: SimpleLineSymbol {
+                        style: Enums.SimpleLineSymbolStyleSolid
+                        color: "black"
+                        width: 2
+                    }
+                }
+            }
+        }
+
+        onMouseClicked: {
+            // Set the attribute prompt to open at the mouse click location
+            attributePromptX = mouse.x;
+            attributePromptY = mouse.y;
+            attributePrompt.visible = true;
+
+            // Create a new feature to append attribute values to
+            newFeature = nestsLayer.featureTable.createFeatureWithAttributes({}, mouse.mapPoint);
+        }
+    }
+
+
+    Control {
+        id: attributePrompt
+
+        // Expand the attributes pane in a direction depending on where the user clicks or taps
+        x: (attributePromptX + width > sampleWindow.height ? attributePromptX - width : attributePromptX) ?? 0;
+        y: (attributePromptY + height > sampleWindow.height ? attributePromptY - height : attributePromptY) ?? 0;
+
+        background: Rectangle {
+            color: "#fdfdfd"
+        }
+
+        visible: false
+
+        contentItem: Column {
+            id: inputColumn
+            spacing: 5
+            padding: 10
+
+            Text {
+                text: "Status"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            ComboBox {
+                id: statusComboBox
+
+                // This ComboBox is hardcoded, but can be dynamically obtained from the feature layer's domain values
+                model: ["", "Occupied", "Unoccupied"]
+
+                onCurrentValueChanged: {
+                    if (statusComboBox.currentValue === "") {
+                        protectionComboBox.model = [""];
+                        return;
+                    }
+
+                    // Update the feature's attribute map with the selection
+                    newFeature.attributes.replaceAttribute("Status", statusValues[statusComboBox.currentValue]);
+
+                    // Append the valid contingent coded values to the subsequent ComboBox
+                    protectionComboBox.model = [""].concat(getContingentValues("Protection", "ProtectionFieldGroup"));
+                }
+            }
+
+            Text {
+                text: "Protection"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            ComboBox {
+                id: protectionComboBox
```

Comment:
The protection field always appears to be unpopulated in the QML sample, whereas in the Cpp sample I have options for "endangered" and "not endangered"

Reply:
That's potentially because the contingentValuesDefinition wasn't loading properly?

Reply:
Yeah, seems to be working as expected now. Thanks